.circleci/config.yml

```diff
@@ -449,16 +440,11 @@ workflows:
           filters: *filter_rebuild_tags
           requires:
             - e2e_tests
-      - anchore/image_scan:
-          name: anchore_image_scan
+      - grype/list-image-vulns:
           context: dockerhub
-          filters: *filter_rebuild_tags
           requires:
             - push_dev_image
-          image_name: anchore/anchore-cli-dev:${CIRCLE_SHA1}
-          private_registry: true
-          timeout: '2000'
-          policy_failure: true
+          image-name: anchore/anchore-cli-dev:${CIRCLE_SHA1}
```

Comment:
This can use ${CIRCLE_TAG} for the image tag.

Reply:
updated. thanks!